examples/bzlmod/hello_world_no_cargo/.bazelrc

```
@@ -0,0 +1,3 @@
+common --experimental_enable_bzlmod
+common --noenable_workspace
+common --enable_runfiles
```

Nit: add a newline (I really need to add a lint for this)
```suggestion
common --enable_runfiles

```